templates/_devcontainer/docker-compose.yml.ts

```diff
@@ -28,29 +27,14 @@ services:
             - LC_ALL=en_US.UTF-8
             - TZ=Europe/Berlin
             - SETGID=1000
-${needsParcel ? (`
-    parcel:
-        container_name: parcel-${adapterNameLowerCase}
-        build: ./parcel
-        expose:
-            - 1234
-        ports:
-            - '1235:1235'
-        volumes:
-            - ..:/workspace:cached
-        environment:
-            - CHOKIDAR_USEPOLLING=1
-`) : ""}
```

Comment:
Why is all this being removed? Same for the other removals

Reply:
Ahm?
you  changed the build system to esbuild and parcel isn’t called anymore
So this Container is useless
https://github.com/ioBroker/create-adapter/blob/master/docs/updates/20220215_esbuild.md
https://github.com/ioBroker/adapter-dev/commit/b6036c6bd32d2f0e713ae5c5b4f3cab60917ce63